tests/widgets/test_graphics_view.py

```diff
@@ -60,7 +60,7 @@ def test_basics_graphics_view():
     assert view.centralWidget is central_item
     # XXX: Removal of central item is not clear in code
     scene = view.sceneObj
-    assert isinstance(scene, pg.GraphicsScene)
+    assert isinstance(scene, pg.GraphicsScene.GraphicsScene)
```

Comment:
Do we have a clear convention for the capitalization of modules?
This one probably does not need to be accessed by the user typically, and thus doesn't justify a change.

But in an ideal world (with no existing user code), should this change to `pg.graphicsscene.GraphicsScene`?
I would argue that a no-caps convention is used by e.g. `pg.functions` and `pg.colormap`.

Reply:
Well, whatever convention there might be, it can't be retroactively applied to modules that have been around such a long time.

Reply:
I'll give you that, although this may be the first time that we are encouraging access outside of the `pg.` namespace.   :) 

I should have really phrased that more openly, though. Sorry about that!

Is there anything we can do to keep module names as consistent as possible *in the future*?
Do you have a preference for the naming?
I do like no-caps for modules, and find e.g. `pg.GraphicsScene.GraphicsScene` a little confusing. But if it turns out that we really like CamelCase names for the modules, then we might still be able to change the module name e.g. for `colormap.py`. All the module level functions are still marked experimental there, but if we wanted to rename it, we probably shouldn't wait any longer.

This shouldn't be any holdup for this PR, though.

Reply:
>although this may be the first time that we are encouraging access outside of the pg. namespace.

That's a question to be answered for ```GraphicsScene```. It was imported into pg namespace only indirectly. Only one place in the library accessed it as ```pg.GraphicsScene```. So I had a choice of either importing it explicitly into pg namespace or modifying the access to pg.GraphicsScene.GraphicsScene. I picked the latter, but if anyone feels that the former would be a better choice, I can make the change.

>Do you have a preference for the naming?

Well, I prefer snake_case. :-) But there's a rule that when modifying code, try to make it look like the rest.

Reply:
@pijyoi regarding snake case wonder if we should try to integrate the `from __feature__ import snake_case`   Not sure what's involved with that level of integration though.